licenses.html

```diff
@@ -4949,6 +4949,7 @@ <h4>Used by:</h4>
                     <li><a href=" https://github.com/RustCrypto/traits ">digest</a></li>
                     <li><a href=" https://github.com/RustCrypto/utils ">opaque-debug</a></li>
                     <li><a href=" https://github.com/RustCrypto/hashes ">sha-1</a></li>
+                    <li><a href=" https://github.com/RustCrypto/hashes ">sha2</a></li>
```

Comment:
Hmmm :thinking: do we need to include licenses for dependencies of our xtask? Probably difficult to filter that out I guess...

Reply:
Uhm I'm not really sure, let me check

Reply:
I don't think we need to.